internal/manifests/datasource/grafana.go

```diff
@@ -0,0 +1,38 @@
+package datasource
+
+import (
+	grafanav1 "github.com/grafana-operator/grafana-operator/v5/api/v1beta1"
+	configv1alpha1 "github.com/grafana/tempo-operator/apis/config/v1alpha1"
+	"github.com/grafana/tempo-operator/internal/manifests/manifestutils"
+)
+
+// Datasource creates a Datasource for Grafana Tempo.
+func Datasource(featureGates configv1alpha1.FeatureGates, namespace string) *grafanav1.GrafanaDatasource {
+	var tlsSkipVerify = true
+
+	if featureGates.OpenShift.ServiceCertsService{
+		tlsSkipVerify = false
+	}
+
+	return &grafanav1.GrafanaDatasource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "tempo",
+			Namespace: namespace,
+		},
+		Spec: grafanav1.GrafanaDatasourceSpec{
+			DatasourceSpec: grafanav1.DatasourceSpec{
+				Access: "proxy",
+				Name:  "Tempo",
+				Type: "tempo",
+				// TODO: Change URL
+				URL:  featureGates.Observability.Datasources.TempoGatewayEndpoint,
```

Comment:
```suggestion
				URL:  fmt.Sprintf("https://%s:%d", naming.ServiceFqdn(params.Tempo.Namespace, params.Tempo.Name, manifestutils.QueryFrontendComponentName), manifestutils.PortHTTPServer),
```

Reply:
Good point! Yes, we should.
I was thinking we support the basic setup now without the gateway (and get this PR merged) and then extend it with support for the gateway in a follow-up PR.

Reply:
Yep, sounds good!